### ISSUE TITLE
The warranty information isn't displayed

### ISSUE THREAD

**Issue by Tommy12pl** [BartoszCichecki/LenovoLegionToolkit#1530]
### Version

2.24.2

### OS

Windows 11 24H2 build 26100.2605

### Device

Lenovo LEGION Y7000P IRX9 (Chinese Mainland)

### BIOS version

NMCN29WW

### What is wrong?

LTT cannot display the warranty information of my laptop.

### What did you try already?

Clicking the refresh button, but it does not work.

### How to reproduce the bug?

1. Open LLT
2. Click "Legion Y7000P IRX9" (Model of my laptop)
3. The Warranty Date isn't displayed

### What is the behavior that you expected?

LTT will display the Warranty Information.

### Logs

[log_2024_12_28_06_37_34.txt](https://github.com/user-attachments/files/18265285/log_2024_12_28_06_37_34.txt)

**Comment by Ace-Radom** [comment]
@Tommy12pl There's a question since I cannot figure out what's wrong with the log you provided. Could you see the warranty info before? If yes, when did it start not to display?

**Comment by BartoszCichecki** [comment]
I am actually thinking of removing the warranty info and leaving just the support link where warranty can be checked.

These issues are just a pain to troubleshoot and it doesn’t really add much value.

**Comment by Tommy12pl** [comment]
> @Tommy12pl There's a question since I cannot figure out what's wrong with the log you provided. Could you see the warranty info before? If yes, since when did it start not to display?

Unfortunately, from the very beginning of using LLT, the warranty information just doesn't show up.

**Comment by Tommy12pl** [comment]
> I am actually thinking of removing the warranty info and leaving just the support link where warranty can be checked.
> 
> These issues are just a pain to troubleshoot and it doesn’t really add much value. 

I think it's right to remove this feature, which is too unstable and takes too much of your time and effort to maintain.
In fact, in mainland China, Lenovo provides a mobile app for checking warranty information, purchasing Lenovo products and providing after-sales service.

**Comment by BartoszCichecki** [comment]
To be fair it's only a problem with Chinese variants, because there are so many different warranty products. But regardless there isn't much value in this.

**Comment by Ace-Radom** [comment]
> Unfortunately, from the very beginning of using LLT, the warranty information just doesn't show up.

Yeah that's what I thought...shows that it's something weird Lenovo did instead of something LLT did wrong.

> To be fair it's only a problem with Chinese variants, because there are so many different warranty products. But regardless there isn't much value in this.

That's true, warranty info isn't so important to show in LLT. Would you consider removing this feature totally for all users, or keeping it for those who have already got the warranty info before and infos have already been stored in settings?

**Comment by BartoszCichecki** [comment]
> Would you consider removing this feature totally for all users, or keeping it for those who have already got the warranty info before and infos have already been stored in settings?

Good question. Since most of the issues are with Chinese warranty specs maybe the best idea is to make it more "dynamic" and only show warranty info if it could be fetched.

There also have been many cases of LLT showing wrong dates for CN models due to complicated warranty products, so maybe just keeping this for global models only would be a good idea?

**Comment by Ace-Radom** [comment]
> Good question. Since most of the issues are with Chinese warranty specs maybe the best idea is to make it more "dynamic" and only show warranty info if it could be fetched.
> 
> There also have been many cases of LLT showing wrong dates for CN models due to complicated warranty products, so maybe just keeping this for global models only would be a good idea?

Sounds good to me. We can also add a msg like `Warranty infos are not available for CN specific models` and add a explaination why this feature has been partly removed in README.
